tests/phpunit/includes/bootstrap.php

```diff
@@ -82,6 +82,12 @@
 	$phpunit_polyfills_autoloader = dirname( dirname( dirname( __DIR__ ) ) ) . '/vendor/yoast/phpunit-polyfills/phpunitpolyfills-autoload.php';
 	$phpunit_polyfills_error      = false;
 
+	// Define WP_TESTS_PHPUNIT_POLYFILLS_PATH if set as environment variable.
+	$phpunit_polyfills_path = getenv( 'WP_TESTS_PHPUNIT_POLYFILLS_PATH' );
+	if ( $phpunit_polyfills_path ) {
+		define( 'WP_TESTS_PHPUNIT_POLYFILLS_PATH', $phpunit_polyfills_path );
```

Comment:
Could we use a different variable name for this ? Maybe `$phpunit_polyfills_path_env`  ?
I'm a bit wary of re-using the same variable name for two scenarios.

Reply:
Done